src/qt/test/compattests.cpp

```diff
@@ -2,11 +2,11 @@
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
-#include "paymentrequestplus.h" // this includes protobuf's port.h which defines its own bswap macos
```

Comment:
The include is being lost here because of the apostophe in the comment. I think replacing `echo '...'` with `echo \"...\"` in the script should fix it.